### PR TITLE
Add Faker rejected reasons

### PR DESCRIPTION
**Pull request by simonwhatley**
See https://trello.com/c/mqkOrAWA/3005-show-feedback-for-rejected-applications-generated-with-faker

### CODE REVIEW

File: app/data/generators/rejection.js [hunk 1]
```diff
@@ -1,11 +1,106 @@
...
+
+  const quality = {
+    'application-quality': 'Yes',
+    'application-quality-reasons': [
+      'Personal statement',
+      'Subject knowledge',
+      'Other'
+    ],
+    'application-quality-reasons-personal-statement': 'Lack of passion',
+    'application-quality-reasons-subject-knowledge': 'Didn’t know enough about maths',
```
Comment (adamsilver) [inline app/data/generators/rejection.js:34]: Did not?

File: app/data/generators/rejection.js [hunk 2]
```diff
@@ -1,11 +1,106 @@
...
+  }
+
+  const qualifications = {
+    'missing-qualifications': 'Yes',
+    'missing-qualifications-reasons': [
+      'No maths GCSE grade 4 (C) or above, or accepted equivalent',
+      'No English GCSE grade 4 (C) or above, or accepted equivalent',
+      'Other'
+    ],
+    'missing-qualifications-reasons-other': 'Another reason goes here'
```
Comment (adamsilver) [inline app/data/generators/rejection.js:46]: Generate a reason with faker or a better one than this.

File: app/data/generators/rejection.js [hunk 3]
```diff
@@ -1,11 +1,106 @@
...
+  const quality = {
+    'application-quality': 'Yes',
+    'application-quality-reasons': [
+      'Personal statement',
+      'Subject knowledge',
+      'Other'
+    ],
+    'application-quality-reasons-personal-statement': 'Lack of passion',
+    'application-quality-reasons-subject-knowledge': 'Didn’t know enough about maths',
+    'application-quality-reasons-other-improve': 'Ways to improve goes here',
```
Comment (adamsilver) [inline app/data/generators/rejection.js:35]: Generate a reason with faker or a better one than this.

File: app/data/generators/rejection.js [hunk 4]
```diff
@@ -1,11 +1,106 @@
...
+      'No maths GCSE grade 4 (C) or above, or accepted equivalent',
+      'No English GCSE grade 4 (C) or above, or accepted equivalent',
+      'Other'
+    ],
+    'missing-qualifications-reasons-other': 'Another reason goes here'
+  }
+
+  const interviewPerformance = {
+    'interview-performance': 'Yes',
+    'interview-performance-advice': 'Advice goes here'
```
Comment (adamsilver) [inline app/data/generators/rejection.js:51]: Generate a reason with faker or a better one than this.

File: app/data/generators/rejection.js [hunk 5]
```diff
@@ -1,11 +1,106 @@
...
+  }
+
+  const course = {
+    'course-full': 'Yes'
+  }
+
+  const offeredOther = {
+    'other-offer': 'Yes',
+    'other-offer-details': 'Other course details go here'
```
Comment (adamsilver) [inline app/data/generators/rejection.js:60]: Generate a reason with faker or a better one than this.

File: app/data/generators/rejection.js [hunk 6]
```diff
@@ -1,11 +1,106 @@
...
+
+  const honesty = {
+    honesty: 'Yes',
+    'honesty-reasons': [
+      'Inaccurate or false information in the application',
+      'Evidence of plagiarism in the application',
+      'References did not support the application',
+      'Other'
+    ],
+    'honesty-reasons-false-information': 'False information detail goes here',
```
Comment (adamsilver) [inline app/data/generators/rejection.js:71]: Generate a reason with faker or a better one than this.

File: app/data/generators/rejection.js [hunk 7]
```diff
@@ -1,11 +1,106 @@
...
+    'honesty-reasons-plagiarism': 'Plagiarism detail goes here',
```
Comment (adamsilver) [inline app/data/generators/rejection.js:72]: Generate a reason with faker or a better one than this.

File: app/data/generators/rejection.js [hunk 8]
```diff
@@ -1,11 +1,106 @@
...
+    'honesty-reasons-reference-information': 'Reference information goes here',
```
Comment (adamsilver) [inline app/data/generators/rejection.js:73]: Generate a reason with faker or a better one than this.

File: app/data/generators/rejection.js [hunk 9]
```diff
@@ -1,11 +1,106 @@
...
+    'honesty-reasons-other': 'Another reason goes here'
```
Comment (adamsilver) [inline app/data/generators/rejection.js:74]: Generate a reason with faker or a better one than this.

File: app/data/generators/rejection.js [hunk 10]
```diff
@@ -1,11 +1,106 @@
...
+  }
+
+  const safeguarding = {
+    safeguarding: 'Yes',
+    'safeguarding-reasons': [
+      'The vetting process found information which makes the candidate unsuitable to work with children'
+    ],
+    'safeguarding-reasons-vetting-information': 'Reasons the candidate unsuitable to work with children'
```
Comment (adamsilver) [inline app/data/generators/rejection.js:82]: Generate a reason with faker or a better one than this.

File: app/data/generators/rejection.js [hunk 11]
```diff
@@ -1,11 +1,106 @@
...
+  }
+
+  const additionalFeedback = {
+    'other-feedback': 'Yes',
+    'other-feedback-details': 'Additional feedback goes here'
```
Comment (adamsilver) [inline app/data/generators/rejection.js:87]: Generate a reason with faker or a better one than this.

File: app/data/generators/rejection.js [hunk 12]
```diff
@@ -1,11 +1,106 @@
...
+  }
+
+  const futureApplications = {
+    'future-applications': 'Yes',
+    'future-applications-details': 'Be happy to consider again'
```
Comment (adamsilver) [inline app/data/generators/rejection.js:92]: There shouldn't be any details for this. Where did you get this from?

**Review by adamsilver — CHANGES_REQUESTED** [review]
Left comments inline.

**Review by simonwhatley — COMMENTED** [review]
Changes made as requested

**Review by adamsilver — APPROVED** [review]

**Pull request merged**